src/dataset.py

```diff
@@ -1,184 +1,123 @@
 from datetime import datetime
-from typing import Optional
+from typing import Tuple
 
 import numpy as np
 import pandas as pd
-import torch
 from torch.utils.data import Dataset
 
-from .dataops import NUM_TIMESTEPS
+from .dataops import BANDS, S1_S2_ERA5_SRTM, DynamicWorld2020_2021
 from .masking import MaskedExample, MaskParamsNoDw
-from .utils import construct_single_presto_input
 
 
-class WorldCerealDataset(Dataset):
-    def __init__(self, dataframe: pd.DataFrame, mask_params: Optional[MaskParamsNoDw] = None):
+class WorldCerealBase(Dataset):
+    _NODATAVALUE = 65535
+    NUM_TIMESTEPS = 12
```

Comment:
Yes, I agree (about masking). I think this would not impact the pre-training portion of the code, so we can probably tackle that with a mask in the labelled dataset.